Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -616,6 +616,67 @@ extension SPNoteEditorViewController {
     }
 }
 
+// MARK: - Search Map
+//
+extension SPNoteEditorViewController {
+
+    /// Show search map keyword ranges
+    ///
+    @objc
+    func showSearchMap(with searchRangeValues: [NSValue]) {
```

Comment:
Would it be possible to add a note to convert this into `[Range]` (or `[NSRange]` ?)

Reply:
Definitely!